QMLComponents/utilities/appdirs.cpp

```diff
@@ -78,8 +78,8 @@ QString AppDirs::bundledModulesDir()
 {
 	static QString folder;
 #ifdef _WIN32
-	auto env = DynamicRuntimeInfo::getInstance()->getRuntimeEnvironment();
-	bool useAppdata =  env != DynamicRuntimeInfo::MSIX;
+	auto env = DynamicRuntimeInfo::getInstance()->runtimeEnvironment();
+	bool useAppdata =  env != RuntimeEnvironment::MSIX && env != RuntimeEnvironment::ZIP;
```

Comment:
? why did you add this.
Dont we want ZIP stuff to stay in the zip folder as much as possible?
I mean it will work fine but might collide if someone has normal jasp installed as well. 
If we do this we should make some other changes as well